crates/control/src/search_suggestor.rs

```diff
@@ -0,0 +1,175 @@
+use color_eyre::Result;
+use context_attribute::context;
+use framework::{AdditionalOutput, MainOutput};
+use nalgebra::{DMatrix, Isometry2, Point2};
+use serde::{Deserialize, Serialize};
+use types::{
+    ball_position::{BallPosition, HypotheticalBallPosition},
+    field_dimensions::FieldDimensions,
+    parameters::SearchSuggestorParameters,
+};
+
+#[derive(Deserialize, Serialize)]
+pub struct SearchSuggestor {
+    heatmap_dimensions: (usize, usize),
```

Comment:
Maybe don't save this in the struct but use `self.heatmap.shape()` instead

Reply:
Done